docs/tutorials/python/tutorial_scripts/dataset.py

```diff
@@ -17,9 +17,11 @@
 syn = Synapse()
 syn.login()
 
-project = Project(name="My Testing Project").get()  # Replace with your project name
+project = Project(
+    name="My uniquely named project about Alzheimer's Disease"
+).get()  # Replace with your project name
```

Comment:
The lines for the dataset tutorial now need to be shifted down by 2

Reply:
Yup, I know. Just haven't pushed the change yet

Reply:
Gotcha! no worries :D